src/image_analyzer.py

```diff
@@ -0,0 +1,16 @@
+import io
+
+from PIL import Image
+from PIL import ImageStat
```

Comment:
You can move this import to the previous line

Reply:
done